### ISSUE TITLE
New release breaks on python 3.5

### ISSUE THREAD

**Issue by mtreinish** [PyO3/setuptools-rust#85]
The new release of setuptools-rust isn't compatible with python 3.5 since it uses fstrings. This is fine, especially since 3.5 is EoL now, but there is no minimum python version set in the package metadata. This means that people trying to use python 3.5 with setuptools-rust it will have pip install the new package version and fail. For example:

https://github.com/Qiskit/retworkx/pull/189/checks?check_run_id=1347529773#step:6:30

It would be good if the next release marked that the minimum supported version was 3.6.

**Comment by mtreinish** [comment]
I had a similar issue on some packages I maintained when I dropped 2.7 support. The only way I can think of to do this gracefully is to push out a 0.11.5 that removes the fstring usage and works with 3.5 so pip will prefer that over 0.11.4. Then push out 0.11.6 which sets the `python_requires=">=3.6"` in the package metadata.

Or the alternative is to just assume anyone still supporting python 3.5 will set a cap like I did in Qiskit/retworkx#190 and push out a 0.11.5 with `python_requires=">=3.6"`. Although looking at: https://pypistats.org/packages/setuptools-rust I'm kinda surprised how many downloads are still on python 3.5.
